src/compo/tropomi_no2_nc2ioda.py

```diff
@@ -60,110 +52,134 @@ def __init__(self, filenames):
     # Open input file and read relevant info
     def _read(self):
         # set up variable names for IODA
-        for iodavar in ['nitrogen_dioxide_in_tropospheric_column', 'nitrogen_dioxide_in_total_column']:
-            self.varDict[iodavar]['valKey'] = iodavar, iconv.OvalName()
-            self.varDict[iodavar]['errKey'] = iodavar, iconv.OerrName()
-            self.varDict[iodavar]['qcKey'] = iodavar, iconv.OqcName()
-            self.varAttrs[iodavar, iconv.OvalName()]['coordinates'] = 'longitude latitude'
-            self.varAttrs[iodavar, iconv.OerrName()]['coordinates'] = 'longitude latitude'
-            self.varAttrs[iodavar, iconv.OqcName()]['coordinates'] = 'longitude latitude'
-            self.varAttrs[iodavar, iconv.OvalName()]['units'] = 'mol m-2'
-            self.varAttrs[iodavar, iconv.OerrName()]['units'] = 'mol m-2'
+        if self.columnType == 'total':
```

Comment:
Since you're passing this through in obsVar, do you need this if statement here?

Reply:
yes as what is passed is the dictionary defined in main().

Reply:
or do you mean total should be as default if no -c/--column option ?

Reply:
instead of passing a dict, since you are setting it in an if, I would send it like so:
```
obsVar = {'inputvar': 'nitrogendioxide_total_column',
                    'iodavar': 'nitrogen_dioxide_in_total_column',
    }
```
so then you don't need the if, elif here, just use the keys

Reply:
`iodavar = self.obsVar['inputvar']`

Reply:
But I see how that complicates the loop below, so I guess it's fine

Reply:
I decided to keep that loop as I think we can still use the dictionary in case we need multiple obsvar in the ioda file.
But this might be irrelevant in the future. I'd just like to keep this feature for now, just in case, before we start generalizing this converter to other tropoomi and omi species or KNMI retrieval products in general, as I assume they are providing the same nc file template.